app/src/main/java/com/kkolontay/theworld/api/ApiInterface.kt

```diff
@@ -0,0 +1,16 @@
+package com.kkolontay.theworld.api
+
+import com.kkolontay.theworld.model.Country
+import retrofit2.http.GET
+
+interface ApiInterface {
+        @GET("all")
```

Comment:
should be:

`suspend fun getAllCountries(): Response<List<Country>>`

Reply:
done